pkg/minikube/proxy/proxy.go

```diff
@@ -44,6 +44,9 @@ func isInBlock(ip string, block string) (bool, error) {
 	if i == nil {
 		return false, fmt.Errorf("parsed IP is nil")
 	}
+	if !strings.Contains(block, "/") {
```

Comment:
Should this return `true` if `ip == block`, in this case?

Reply:
I see.
NO_PROXY can be set ip(XX.XX.XX.XX) and cidr(XX.XX.XX.XX/XX).
It's case should be added.

I added the case(`ip == block`). And I changed to execute ParseCIDR only when the block is cidr.
Thanks!